### DISCUSSION TITLE
How does it compare to FlexGet?

### DISCUSSION THREAD

**Discussion by iwinux** [autobrr/autobrr#567] [Q&A]
New to this torrent automation territory, I'm still surveying all the options available. If I read it right, autobrr and FlexGet is the same kind of tool, right? The README also mentions FlexGet as the source of inspiration. So I'd like to ask: what factors should I consider when deciding between autobrr v.s. FlexGET?

**Comment by Yankees4life** [comment]
Compared to Flexget, you get a nice webui...and it isn't complex to set up
